packages/lesswrong/server/fmCrosspost/routes.ts

```diff
@@ -76,17 +76,18 @@ export const addCrosspostRoutes = (app: Application) => {
       } catch (e) {
           // eslint-disable-next-line no-console
         console.error('Error when making cross-site GET request', { route: route.path, error: e });
+        const errorCode = e instanceof ApiError ? e.code : 501;
         return res
-          .status(e instanceof ApiError ? e.code : 501)
-          .send({error: e.message ?? "An unknown error occurred"})
+          .status(200)
```

Comment:
Can we add a comment here (and maybe below as well). If I was editing some code around here and saw that we return status 200 along with an error message I'd probably automatically change it to 400 without thinking.